nx.json

```diff
@@ -18,17 +18,22 @@
         "^compile"
       ],
       "inputs": [
-        "{projectRoot}/src",
-        "{projectRoot}/test"
+        "{projectRoot}/src/**/*.ts",
```

Comment:
note to reviewer: although `nx` documentation says using folder names is okay I found this option to be more sensitive to changes.